fe/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -540,7 +541,7 @@ public void readFields(DataInput in) throws IOException {
         txIsolation = Text.readString(in);
         autoCommit = in.readBoolean();
         resourceGroup = Text.readString(in);
```

Comment:
You need to upgrade meta version if you change the meta data log format.
Set FeMetaVersion.java to more details

Reply:
fixed